### PR TITLE
ncm-syslog: Expand facility regex to allow comma seperated lists

### PR DESCRIPTION
**Pull request by jrha**
Fixes problem reported in 17.7.0-rc1 (quattor/release#300).

### CODE REVIEW

**Comment by stdweird** [comment]
add a unittest?

**Comment by jrha** [comment]
Yes!

File: ncm-syslog/src/test/perl/configure.t [hunk 1]
```diff
@@ -43,6 +43,7 @@ random stuff
 *.* other action
 user.crit\tawesome
 mail.debug\tawesome
+uucp,news.crit\t/var/log/spooler
```
Comment (stdweird) [inline ncm-syslog/src/test/perl/configure.t]: need to add this line either to orig in the next tests, or always to the expected value

**Comment by jrha** [comment]
@stdweird - better?

**Pull request merged**